content/docs/start/data-management/metrics-parameters-plots.md

```diff
@@ -21,16 +21,21 @@ DVC makes it easy to track [metrics](/doc/command-reference/metrics), update
 <admon type="tip">
 
 All of the above can be combined into [experiments] to run and compare many
-iterations of your ML project.
+iterations of your machine learning project.
```

Comment:
minor: what do we use in other place? It might have been done for SEO (for me it's def easier to read ML at this point)

Reply:
we do both:
- "ML" - appears 113 times in `md` files in this repo
- "machine learning" - appears 44 times in `md` files in this repo

> It might have been done for SEO

Can you please explain what you mean by that? 
Is "ML" better than "machine learning" for SEO? or the other way around? or there's some benefit in balancing appearances? honestly asking, "for SEO" is often used very vaguely.

Regardless - especially in this context, no opinion either way, so if it bothers you - changing back to ML 👍

Reply:
We did analysis of how Google and other search engines see us, which keywords we should highlight more (in titles for example, on pages, etc) (from brief look in the history https://github.com/iterative/dvc.org/pull/1806/files ) . I can try to find some docs for this. My most likely, ppl were trying to keep ML _and_ machine learning on the same page. 

Overall, it's minor and not very sustainable to bother about this to this level (very hard to share the knowledge about this as we see, etc). 

In this case take it as "it's just easier to read as ML" (my personal take). But I would also try to avoid changes like this at all.